plugin/web/extensions/sidebar/core/components/hooks.ts

```diff
@@ -64,6 +64,8 @@ export default () => {
   const [fieldTemplates, setFieldTemplates] = useState<Template[]>([]);
   const [project, updateProject] = useState<Project>(defaultProject);
   const [selectedDatasets, setSelectedDatasets] = useState<DataCatalogItem[]>([]);
+  const [savingTemplate, setSaveTemplate] = useState<boolean>(false);
+  const [savingDataset, setSaveDataset] = useState<boolean>(false);
```

Comment:
Is it possible just to have a single `loading` that is used for both template and dataset?

Reply:
done

Reply:
8f47d1a450a11db460d7b822d7ed03fa51da839d

Reply:
2b0e25c711b0332eadd8ede255bf42f49778bfca